cfme/containers/provider/openshift.py

```diff
@@ -77,10 +71,12 @@ def from_config(prov_config, prov_key, appliance=None):
             credentials={'token': token_creds},
             key=prov_key,
             zone=prov_config['server_zone'],
-            hostname=prov_config.get('hostname', None) or prov_config['ip_address'],
-            port=prov_config['port'],
-            sec_protocol=prov_config.get('sec_protocol', None),
-            hawkular_sec_protocol=prov_config.get('hawkular_sec_protocol'),
+            hostname=prov_config['endpoints']['default'].hostname or prov_config['ipaddress'],
```

Comment:
nevermind, it will concern you a bit later when container providers are moved to Endpoints approach along with widgetastic conversion.